test/jdk/java/lang/ClassLoader/Assert.java

```diff
@@ -62,7 +62,7 @@ public static void main(String[] args) throws Exception {
         int[] switches = new int[7];
 
         int switchSource = 0;
-        if (args.length == 0) { // This is master version
+        if (args.length == 0) { // This is the coordinator version
```

Comment:
Perhaps s/coordinator/controller/?

Reply:
Let's change it to: 

// This is the controller

Reply:
Will do.